lightautoml/text/sentence_pooling.py

```diff
@@ -1,5 +1,5 @@
 """Different Pooling strategies for sequence data."""
-
+# flake8: noqa
```

Comment:
set exact rule to ignore

Reply:
Specified error codes in the configuration file.